.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
please take the API version of that file

Reply:
Fixed